### PR TITLE
Updated the Brew mesheryctl installation instructions #343

### LINKED ISSUE #343: Update Brew mesheryctl installation instructions
**Current State:**
Currently, in the Getting Started page for Meshery (https://meshery.io/#getting-started), the Brew installation instructions are outdated. When you run them, you'll get an error since `mesheryctl` is not properly installed.

**Desired State:**
The instructions should be updated based on the instructions under the "Installing with Homebrew section" here: https://meshery.layer5.io/docs/installation

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Fahdmoh01**
Signed-off-by: Fahdmoh01 <fahdmoh.1@gmail.com>

**Description**
Updated the Brew mesheryctl installation instructions
This PR fixes  #343

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [*] Yes, I signed my commits.

### CODE REVIEW

**Comment by Fahdmoh01** [comment]
@Nikhil-Ladha

File: _includes/getting-started.html [hunk 1]
```diff
@@ -66,11 +66,13 @@ <h3>Brew User:</h3>
             <p>Install on Mac or Linux using Homebrew:</p>
 
             <div id="start-brew" class="highlight-code">
-              <p class="code-box" style="height: 60px; line-height: 30px;"><code style="position: absolute;"> $ brew install layer5io/mesheryctl
+              <p class="code-box" style="height: 85px; line-height: 30px;"><code style="position: absolute;"> $ brew tap layer5io/tap
```
Comment (Nikhil-Ladha) [inline _includes/getting-started.html:69]: Increase the height to 95px to avoid the scrollbar.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: _includes/getting-started.html (hunk 1 above)
Comment (Fahdmoh01) [reply _includes/getting-started.html:69]: okay

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you @Fahdmoh01 :+1:

**Comment by Fahdmoh01** [comment]
You are welcome

**Comment by leecalcote** [comment]
Good work, @Fahdmoh01. 😄

**Pull request merged**